internal/config/config.go

```diff
@@ -72,13 +73,15 @@ func Read(path string, cfg interface{}) (err error) {
 		}
 		err = f.Close()
 	}()
-	switch filepath.Ext(path) {
-	case ".yaml":
+	switch ext := filepath.Ext(path); ext {
+	case ".yaml", ".yml":
 		err = yaml.NewDecoder(f).Decode(cfg)
 	case ".json":
 		err = json.Decode(f, cfg)
+	default:
+		return errors.ErrUnsupportedConfigFileType(ext)
 	}
-	return err
+	return nil
```

Comment:
why do we return nil here? 🤔

Reply:
I think this statement is unreachable.

Reply:
then do we need to change line 78 & line 80 to return?

Reply:
I think either will work.